package-lock.json

```diff
@@ -1,11600 +1,8 @@
 {
   "name": "webrecorder",
   "version": "0.0.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Make sure not to change this file. You might have old npm or node installed.

Reply:
@YukiChung, easiest way to fix this issue is to remove the file in your branch and then copy the same file from main branch to replace the file. After you push, this will not be assumed as a change.

Reply:
Got it~